### PR TITLE
Web app demo

### PR DESCRIPTION
**Pull request by goncalo-rodrigues**

### CODE REVIEW

File: blog/2022-08-29-web-app-demo/index.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+<!--truncate-->
+
+
+## Architecture
+
+This web app will contain the following components:
+
+- Networking - a virtual network that can be accessed from the internet with the appropriate security group
+- Database - a managed mysql database where data will be stored
```
Comment (danvdb18) [inline blog/2022-08-29-web-app-demo/index.md]: mysql -> MySQL

File: blog/2022-08-29-web-app-demo/index.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+<!--truncate-->
+
+
+## Architecture
+
+This web app will contain the following components:
+
+- Networking - a virtual network that can be accessed from the internet with the appropriate security group
+- Database - a managed mysql database where data will be stored 
+- Virtual machine - a server configured to serve a sample notes app that stores and retrieves data from the database
```
Comment (danvdb18) [inline blog/2022-08-29-web-app-demo/index.md]: sample -> simple

File: blog/2022-08-29-web-app-demo/index.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+terraform {
+  required_providers {
+    multy = {
+      source = "multycloud/multy"
+    }
+  }
+}
+
+provider "multy" {
+  # get a free api key at https://multy.dev/#beta
```
Comment (danvdb18) [inline blog/2022-08-29-web-app-demo/index.md]: This kind of gets lost here. Would include it in the para above when you're talking about passing the cloud credentials.

File: blog/2022-08-29-web-app-demo/index.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+resource "multy_route_table_association" "rta" {
+  route_table_id = multy_route_table.rt.id
+  subnet_id      = multy_subnet.subnet.id
+}
+
+```
+
+## Configuring database
+
+Next up, we need to configure our database. We want a small mysql database with a random password in the same location where we placed our virtual network.
```
Comment (danvdb18) [inline blog/2022-08-29-web-app-demo/index.md]: mysql -> MySQL

File: blog/2022-08-29-web-app-demo/index.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+}
+
+output "endpoint" {
+  value = "http://${multy_virtual_machine.vm.public_ip}:4000"
+}
+```
+
+## Deploying
+
+After you have everything written, you can deploy the resources using terraform. Run the following command, using the cloud and [location](https://docs.multy.dev/regions) you prefer:
```
Comment (danvdb18) [inline blog/2022-08-29-web-app-demo/index.md]: terraform -> Terraform

File: blog/2022-08-29-web-app-demo/index.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+
+```bash
+terraform apply -var="cloud=aws" -var="location=eu_west_1"
+```
+
+After a few minutes, you should see an endpoint where you can access the example app!
+
+## Summary
+
+Terraform is a very powerful tool that makes deployments repeatable using a single unified language (HCL) for any cloud. However, clouds are quite different from each other which means Terraform files look very different from cloud to cloud. In this tutorial we looked at an example on how to leverage Multy to make deployments portable across multiple cloud providers. If you want to explore other examples further, take a look at [Multy examples](https://docs.multy.dev/examples/).
```
Comment (danvdb18) [inline blog/2022-08-29-web-app-demo/index.md]: When you say make deployments portable across multiple cloud providers.... Would it be worth it to say just changing your cloud parameter?

Comment (goncalo-rodrigues) [reply blog/2022-08-29-web-app-demo/index.md]: Yea, good point

**Pull request merged**